CourseApp.Tests/CourseApp.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
please take a look on https://github.com/ISUCT/Tprogramming_2021/pull/5/files MR to fix issue with github actions

Reply:
fixed all troubles with project settings